invenio_assets/assets/package.json

```diff
@@ -13,60 +13,60 @@
     "tinymce": "^6.7.2"
   },
   "devDependencies": {
-    "browserify-zlib": "^0.2.0",
-    "https-browserify": "^1.0.0",
-    "stream-browserify": "^3.0.0",
-    "stream-http": "^3.2.0",
-    "patch-package": "^6.5.0",
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.0",
+    "clean-webpack-plugin": "^4.0.0",
+    "copy-webpack-plugin": "^11.0.0",
+    "css-minimizer-webpack-plugin": "^4.2.0",
+    "mini-css-extract-plugin": "^2.0.0",
+    "sass-loader": "^13.0.0",
+    "terser-webpack-plugin": "^5.0.0",
+    "webpack": "^5.0.0",
+    "webpack-cli": "^5.0.0",
+    "webpack-dev-middleware": "^6.0.0",
+    "webpack-hot-middleware": "^2.24.0",
+    "webpack-livereload-plugin": "^3.0.0",
+    "webpack-merge": "^5.1.0",
+    "webpack-yam-plugin": "^1.0.0",
+
     "@babel/core": "^7.18.0",
-    "ajv": "^8.12.0",
     "@babel/eslint-parser": "^7.18.0",
     "@babel/plugin-proposal-class-properties": "^7.18.0",
-    "@babel/plugin-proposal-private-property-in-object": "^7.21.0",
     "@babel/plugin-transform-runtime": "^7.18.0",
     "@babel/preset-env": "^7.18.0",
     "@babel/preset-react": "^7.18.0",
     "@babel/register": "^7.18.0",
     "@babel/runtime": "^7.18.0",
     "@inveniosoftware/eslint-config-invenio": "^2.0.0",
+    "ajv": "^8.12.0",
     "autoprefixer": "^10.4.0",
     "babel-loader": "^9.0.0",
+    "browserify-zlib": "^0.2.0",
     "chalk": "^5.0.0",
-    "clean-webpack-plugin": "^4.0.0",
     "css-loader": "^6.0.0",
-    "css-minimizer-webpack-plugin": "^4.2.0",
-    "eslint-config-react-app": "^7.0.1",
```

Comment:
this guy caused some issues with the `pnpm + webpack` build (but not for `npm + webpack`) due to some apparent conflicts in `.eslintrc` configs [1] – the simplest "fix" was to remove this dependency... not sure if that's the best way to go, but it did make the builds run through successfully

[1] unfortunately i don't have the error logs at hand anymore, but they should pop up again if this dep gets added back